src/taco-utils/cordovaWrapper.ts

```diff
@@ -10,28 +10,34 @@
 /// <reference path="../typings/node.d.ts" />
 /// <reference path="../typings/Q.d.ts" />
 /// <reference path="../typings/semver.d.ts" />
+/// <reference path="../typings/tacoLiveReload.d.ts" />
+
 
 "use strict";
 
-import assert = require ("assert");
-import child_process = require ("child_process");
-import path = require ("path");
-import Q = require ("q");
-import semver = require ("semver");
-import util = require ("util");
+import assert = require("assert");
+import child_process = require("child_process");
+import path = require("path");
+import Q = require("q");
+import semver = require("semver");
+import util = require("util");
+import livereload = require("taco-livereload");
```

Comment:
Yup, for those wondering: I suspect that with npm 3+, the new "flat" structure was causing taco-livereload to be installed at the root of node_modules (because taco-cli has a dependency on it), so taco-utils was able to require() it successfully even if it didn't have it as a dependency. However, for earlier nodes that still use the "nested" structure for dependency, taco-utils is not able to require() taco-cli's taco-livereload